app/routers/books.py

```diff
@@ -57,3 +56,61 @@ async def create_book(book: BookCreateModel, session=Depends(get_session)):
 
     return BookModel.from_orm(book)
 
+
+@router.patch("/{book_id}", response_model=BookModel, status_code=200)
+async def update_book(book_id: int, book: BookCreateModel, session=Depends(get_session)):
+    book = Book(**book.dict(exclude_unset=True))
+    try:
+        item = session.query(Book).filter(Book.id == book_id).first()
+    except SQLAlchemyError as e:
+        print(e)
+        return {"message": "error while working with database"}
+    else:
+        if item:
+            item.title = book.title
+            item.author = book.author
+            session.commit()
+            return BookModel.from_orm(item)
+        else:
+            raise HTTPException(
+                status_code=status.HTTP_204_NO_CONTENT,
```

Comment:
404 should be returned there. 204 is HTTP status code for the successful deletion